manga_translator/streamlit/pages/page1.py

```
@@ -0,0 +1,25 @@
+
+def on_slide_change():
+    count = st.session_state.get("slide_change_count", 0)
+    count += 1
+    st.session_state["slide_change_count"] = count
+    st.write(f"slide changed {count}")
+
+c=st.slider("c",1,10,3, on_change=on_slide_change)
+
+st.write(_src.cached_compute(a, c))
+
+st.session_state
```

Ruff

Found useless expression. Either assign it to a variable or remove it.

(B018)